### ISSUE TITLE
buggy

### ISSUE THREAD

**Issue by diynevala** [LeonardMeagher2/github_to_itch#6]
1. installing plugin conflicts with project README.md
2. enabling plugin opens the config dialog
    - Has a field for itch username and
    - a field for itch project name
    - if you switch to another window like browser to copy the project name, dialog will close
    - I have no idea how to open the dialog again except uninstall plugin and enabling again
    - dialog does not have any other buttons or fields to confirm/save/proceed

Basically, a huge waste of time when nothing works. Using Godot 4.2, maybe update this plugin's README to warn about possible compatibility issue.

There's also a typo in **Tempaltes** -folder name.
